test_variables.py

```diff
@@ -84,7 +87,7 @@ def test(variables):
 
 def test_invalid_format(testdir, file_format):
     testdir.makepyfile("def test(variables): pass")
-    result = run(testdir, file_format, ["invalid"], raw=True)
+    result = run(testdir, file_format, ["invalid="], raw=True)
```

Comment:
Out of curiosity, why was this change needed?

Reply:
> Out of curiosity, why was this change needed?

This was needed because of some strangeness with TOML. A file containing something that looks like a key but with no equals sign or newline returns an empty dict, not a ValueError. It happens in at least two different python toml implementations so i'm guessing it's part of the spec.